app/controllers/pages_controller.rb

```diff
@@ -0,0 +1,7 @@
+class PagesController < ApplicationController
+  def home
```

Comment:
Why `PagesController < ApplicationController`

Reply:
Do you remember previously I asked you why all model < ApplicationModel?

Reply:
I thought the answer last time were not very accurate? Is it that the `ApplicationController` contains built-in code that can be run in all inherit controllers?

Reply:
😅 
Rails motto is by convention. Not configuration. This means there is a set of rules you follow. Good and bad. Bad is that customization can be very hard. Good is that if you know rails, any repo that uses rails, you can know where things are and how things work. Cause they all work the same. Why is this good?

When you go from one team to another or even from one company to another, as long as you hear it's rails, you will be right at home.

So, the convention is to have all controllers inherit from this base class and not from rails base class. Why? Cause rails base class can change in the new version of rails. If it does, you just need to change it in ApplicationController instead of every controller. This works the same for model